src/threadlist.cpp

```diff
@@ -854,10 +859,8 @@ restarthread(void *threadv)
 #ifdef HAS_PR_SET_PTRACER
       prctl(PR_SET_PTRACER, PR_SET_PTRACER_ANY, 0, 0, 0); // For: gdb attach
 #endif // ifdef HAS_PR_SET_PTRACER
-      struct timespec delay = { 15, 0 }; /* 15 seconds */
-      printf("Pausing 15 seconds. Do:  gdb <PROGNAME> %d\n",
-             dmtcp_virtual_to_real_pid(getpid()));
-      nanosleep(&delay, NULL);
+      int dummy = 1;
+      while (dummy);
```

Comment:
An informative print here too. It's good to inform the user that we are waiting for gdb to be attached.